### PR TITLE
[FIX] Show only email apps for sending email

### LINKED ISSUE #2038: [IMPROVEMENT] Setting -> "Contact us" should show only email apps
## Describe the bug
In  `Setting -> Contact` us is meant to send an email at `support@rocket.chat` and it should show only email apps but right now it's showing other apps like WhatsApp etc. which is not the correct behaviour.

### PR DESCRIPTION
**Pull request by divyanshub024**
@RocketChat/android

Closes #2038 

#### Description:
Shows only email apps for sending "contact us" mail.

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt [hunk 1]
```diff
@@ -128,11 +129,13 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
...
     private fun contactSupport() {
-        with(Intent(Intent.ACTION_SEND)) {
-            type = "message/rfc822"
-            putExtra(Intent.EXTRA_EMAIL, arrayOf("support@rocket.chat"))
-            putExtra(Intent.EXTRA_SUBJECT, getString(R.string.msg_android_app_support))
-            putExtra(Intent.EXTRA_TEXT, getDeviceAndAppInformation())
+        val uriText = "mailto:${"support@rocket.chat"}" +
+                "?subject=" + Uri.encode(getString(R.string.msg_android_app_support)) +
+                "&body=" + Uri.encode(getDeviceAndAppInformation())
+        val uri = Uri.parse(uriText)
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt]: You can remove this variable here (`val uri = Uri.parse(uriText)`)...

File: app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt [hunk 2]
```diff
@@ -128,11 +129,13 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
...
+
+        with(Intent(Intent.ACTION_SENDTO)) {
+            data = uri
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt]: ... And define `data` like:

`data = uriText.toUri()`

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by divyanshub024** [comment]
@filipedelimabrito I have made the requested changes.

**Review by philipbrito — APPROVED** [review]
Thanks for sending this PR @divyanshub024 💯

**Pull request merged**